internal/crypto/cert.go

```diff
@@ -22,7 +23,15 @@ const ClientBootstrapCommonNamePrefix = "client-enrollment-"
 const AdminCommonName = "flightctl-admin"
 const DeviceCommonNamePrefix = "device:"
 
+var Prefixes = [...]string{ClientBootstrapCommonNamePrefix, DeviceCommonNamePrefix}
```

Comment:
question: does this need to be exported?

Reply:
Now - now. 

Gut feeling is that it will be needed in the CA shims as there are differences in the way a proper CA handles renew and first issue.